frame/elections/src/lib.rs

```diff
@@ -147,6 +145,8 @@ type ApprovalFlag = u32;
 const APPROVAL_FLAG_LEN: usize = 32;
 
 pub trait Trait: frame_system::Trait {
+	type ModuleId: Get<LockIdentifier>;
```

Comment:
Docs.

Reply:
Added a doc comment ```/// Identifier for the elections-phragmen pallet's lock``` and also an equivalent doc comment for ```elections```.